ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/AttestationUtil.java

```diff
@@ -161,43 +196,58 @@ public static UnsignedLong get_current_epoch_boundary_attesting_balance(BeaconSt
   public static UnsignedLong get_previous_epoch_boundary_attesting_balance(BeaconState state)
       throws Exception {
 
+    List<Integer> previous_epoch_boundary_attester_indices =
+        get_previous_epoch_boundary_attester_indices(state);
+
+    return AttestationUtil.get_total_attesting_balance(
+        state, previous_epoch_boundary_attester_indices);
+  }
+
+  public static List<PendingAttestation> get_previous_epoch_head_attestations(BeaconState state)
+      throws Exception {
     // Get previous epoch
     UnsignedLong previous_epoch = BeaconStateUtil.get_previous_epoch(state);
 
-    // Get previous_epoch_attestations
+    // Get current_epoch_attestations
     List<PendingAttestation> previous_epoch_attestations =
         AttestationUtil.get_epoch_attestations(state, previous_epoch);
```

Comment:
You don't need to prefix the call to `get_epoch_attestations ` with `AttestationUtil` since this code is in `AttestationUtil`.